### PR TITLE
Updating config to require a single approval

### PR DESCRIPTION
**Pull request by jlaswell**

### CODE REVIEW

**Comment by bkuhl** [comment]
This also includes a `Quote` model, is that expected?

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvals = 1
+pattern = "(?i):shipit:|:\\+1:|LGTM"
```
Comment (bkuhl) [inline .lgtm]: Does case matter here?  We should probably add a comment explaining what this does

**Comment by jlaswell** [comment]
@bkuhl I can't quite figure out how to remove that actually...

**Comment by jlaswell** [comment]
@bkuhl @millidavids Should have to put an actual LGTM for this one, and then we can +1

**Comment by bkuhl** [comment]
👍

**Comment by bkuhl** [comment]
lgtm

**Comment by millidavids** [comment]
+1

**Pull request merged**